src/components/Footer/Footer.js

```diff
@@ -22,6 +21,7 @@ import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {COLORS, FONTS} from '../../utils/theme';
 import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
 import {IcDmAnon} from '../../assets/icons/ic_dm_anon';
+import IconCommentArrow from '../../assets/icon/IconCommentArrow';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new SVG component `IconCommentArrow` is imported. This seems to be a part of the redesign and will likely be used in the comment section of the feed.
```diff
+ import IconCommentArrow from '../../assets/icon/IconCommentArrow';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->